tests/ai_safety/evalhub/utils.py

```
@@ -1332,3 +1332,244 @@ def fetch_evalhub_job_logs_while_running(
+        Pod.get(
+            client=admin_client,
+            namespace=operator_ns,
+            label_selector=OPERATOR_POD_LABEL_SELECTOR,
+        )
+    )
+    assert pods, "No operator pod found"
+    pod = pods[0]
+    response = requests.get(
+        f"https://{pod.instance.status.podIP}:{OPERATOR_METRICS_PORT}/metrics",
+        headers={"Authorization": f"Bearer {operator_metrics_token}"},
+        verify=False,
```

Ruff (0.16.1)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)